Sources/Sentry/SentryFileIOTrackingIntegration.m

```diff
@@ -30,8 +30,8 @@ - (BOOL)installWithOptions:(SentryOptions *)options
 
 - (SentryIntegrationOption)integrationOptions
 {
-    return kIntegrationOptionEnableSwizzling | kIntegrationOptionIsTracingEnabled
-        | kIntegrationOptionEnableAutoPerformanceTracing | kIntegrationOptionEnableFileIOTracing;
+    return kIntegrationOptionIsTracingEnabled | kIntegrationOptionEnableAutoPerformanceTracing
```

Comment:
We should allow manual file io tracing without swizzling, see https://github.com/getsentry/sentry-cocoa/pull/4859